spec/requests/relationships_spec.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Relationships', type: :request do
+  describe 'follow user' do
+    image_path = File.join(File.dirname(__FILE__), '..', 'images', 'download.jpeg')
+    let(:follower) { create :user }
+    let(:followed) { create :user }
+
+    let(:relationship) { Relationship.create(follower:, followed:) }
```

Comment:
There is no need to establish a relationship before to test the correct functionality for creation.